### ISSUE TITLE
Internal Analytics Reporting

### ISSUE THREAD

**Issue by adonahue** [rifflearning/zenhub#13]
As a PM, I want to understood how the Riff product is used, so that I have data about what users do. This is important for understanding product use and for how use (or non use) relates to our business objectives. 

Brightscout has recommended two analytics tools that can connect to a database, help you build queries, and visualize the results in a dashboard. This reporting would be used internally (e.g. by product managers, project managers, course TAs) to quantify user behavior patterns and report usage statistics back the the business and to clients.

This story is to allocate time to the following tasks.

- Evaluate Redash (https://redash.io/) and metabase (https://metabase.com/) as candidates for an analytics tool to be used with Riff Platform and Riff Edu. 
- Consider the following scenarios for data analysis: https://docs.google.com/document/d/12QH1hBmoPu6nhS6JgOnbDBeBohRTSOY3UJXWD5htk9c/edit
- A written summary of pros and cons to each product, and recommendation of which to use. 

This story is a duplicate of the Trello card tracking the same work. https://trello.com/c/oQi4hbGN

**Comment by juliariffgit** [comment]
Has made progress on some of these that were straightforward. Working on getting more info about ones that were more vague. Brec and Mike to discuss strategy for deployment. Meeting set for 5/3 to discuss further.

Currently blocked by product design + deployment questions

**Comment by adonahue** [comment]
@brecriffs - I think this portion of the work is largely done. Maybe get the repo set up and check your code in, and I think that will be enough to close this out. Does that make sense to you?

Also, please stick an estimate on here for the work you've done. Thx!

**Comment by juliariffgit** [comment]
Amy added a few new stories to represent additional work we need to do for this.

Brec learned about Docker since last standup, and is getting the Docker composed. Has learned enough to accomplish his tasks.

Plans to finish Docker today and spend tomorrow and Friday wrapping metabase in the Docker container, then will look at new requirements added by Amy. Brec should put estimates on these and add details when he can.

Mike wrote a small compose Brec can use, too.

**Comment by juliariffgit** [comment]
Reports are functional at this time for Brec to do. Scope looks like it's going to change, so we need to decide how much more work we're going to do.

We still need to test how to export this data.

But this particular story is considered done because it was around tool research and rec. Additional work will be added to new stories.
